docs/en/lifecycle.rst

```diff
@@ -0,0 +1,252 @@
+  * **Wallet Instance Revocation** (**WI REV**): for technical security reasons or triggered by external entities (e.g., Users and Supervisory Bodies) the Wallet Cryptographic Hardware Key Tag MUST be deleted from the User account.
+  * **Data Purging**: through an explicit request of Users, the User account at the Wallet Provider MUST be removed from the local storage.
+
+
+Digital Credential
+------------------
+:numref:`fig_DigitalCredential_States` shows the states and transitions for Digital Credentials. 
+It includes four distinct states: **Issued**, **Valid**, **Expired**, and **Revoked**. While, in case of (Q)EAAs there is an additional state: **Suspended**. 
+A Digital Credential in all states can be deleted (**PID/(Q)EAA DEL**) and this ends its lifecycle.
+
+.. _fig_DigitalCredential_States:
+.. figure:: ../../images/DigitalCredential_States.svg
+    :figwidth: 100%
+    :align: center
+    :target: https://www.plantuml.com/plantuml/png/RP9HRzCm4CVV_IbEtSC0AIAK5Q4ze4Lh2fK6b6MRa807BxwrLXmxifsDWFZks8udjr7xLF_kVtS_dN9XBDMsRmNPSOQ0RMS7O6XgpJlBbIHMTM0Lt2jhLGkCQwm39wUGPV0H9Meg7ATRJLimTX1SRbs9c8RBZdh8y87smgwKj1N_W_1clbUiBBLOQAsUBfLG6ku5hPkZzKz8MUX_EorVSOatErut4es1UNJxJ1k4McbdQ81A1iB539XMARj3VUYeLI_PPGZ3F8VuEmL1zHPr70EQCjwRr1P6sg53w9GO_2EszIOXFzkweqIj9JvuQBou2HB-7nH2L2EY1cRk1UDp1l2Nn4pLcmubGmOdgrMnoFF8h_5HDPuktvqjpXQHbhyxhXEDwsyqbOPRhcHO_ZnwRKoFxAk-euApe30IK1e2cpaD6Ar702Tv_Zvt3Wx_UFKBCistEvjzWDXu3flrylMBRo_BelWfrrK5168jPVsaQVJHCsu729-c8V-SvA5UnjIJTDtf7kVmt5tTLfjft4NZYIQhhiixE1AEbvk4o-yRGjBAhEzSzB0vQTn-yI8fFf7O5vY4qlAznK326T974a_WBp_HN9PNvCADwrln7m00
+    
+    Digital Credential Lifecycle.
+
+.. note::
+
+  Users MAY present a Digital Credential in any state, it is up to the Relying Party's policy to accept a not Valid Digital Credential. 
+  An example of this scenario is when a Relying Party needs to verify that the User is not a minor. In this case, even if the User presents an 
+  **Issued/Expired/Revoked** or **Suspended** Digital Credential, the age claim is still reliable.
+
+.. note::
+
+  While **Issued**, **Valid**, **Expired**, **Revoked** are explicitly mentioned in the ARF (see Figure 5 of ARF v1.4), 
+  **Suspended** is implicitly present in the text. For clarity and completeness, in this specification we explicitly consider it.
+
+Transition to Issued
+^^^^^^^^^^^^^^^^^^^^
+For the state machine to start, the Wallet Instance MUST be in either the **Operational** or **Valid** state, enabling credentials to be issued to it. 
+The state machine begins with the **Issued** state, when an issuance process is triggered and, as a result, a Digital Credential is issued to the 
+Wallet Instance (**PID/(Q)EAA ISS**). Please refer to the relevant sections for further information about PID and (Q)EAAs issuance process.
```

Comment:
```suggestion
Wallet Instance (**PID/(Q)EAA ISS**). Please refer to the relevant sections for further information about PID and (Q)EAAs issuance process.
```

I admit to get tired to read so many times the statement `Please refer to the relevant sections for further information about`. I ask to use a shorter sentece with a href reference to the relevant section, or delete the entire statement.